hypershift-operator/controllers/controlplane.go

```diff
@@ -101,26 +98,26 @@ func (r *OpenShiftClusterReconciler) ensureControlPlane(ctx context.Context, clu
 	params := hypershiftcp.NewClusterParams()
 	params.Namespace = name
```

Comment:
Just a thought for future consideration... should we use the namespace of the HostedControlPlane resource as the namespace where we place resources? (instead of using the Name of the resource)